openmetadata-ui/src/main/resources/ui/src/components/Users/Users.component.tsx

```diff
@@ -229,7 +229,7 @@ const Users = ({
       return (
         <div className="tw-mt-4 tw-w-full tw-px-3">
           {isDisplayNameEdit ? (
-            <div className="tw-flex tw-justify-between tw-items-center tw-gap-2">
+            <div className="tw-flex tw-flex-col tw-justify-between tw-gap-2">
```

Comment:
@aniketkatkar97 , Can we use antd `<Space>` component here?
https://ant.design/components/space/

Reply:
Made the suggested changes.